cmd/regctl/image.go

```diff
@@ -271,6 +271,10 @@ regctl image mod registry.example.org/repo:v1 --create v1-extended \
 regctl image mod registry.example.org/repo:v1 --create v1-extended \
   --layer-add "dir=path/to/directory"
 
+# append a layer to all platforms using the contents of a directory with a different workdir
+regctl image mod registry.example.org/repo:v1 --create v1-extended \
+  --layer-add "dir=path/to/directory,workdir=/tmp"
```

Comment:
`workdir` is a well known term in Dockerfile's and the use here differs in a few ways. Later steps do not default to this location. The image configuration is not updated with that settings. And the directory, and parent directories, are not created by this.

Reply:
Thank you for your valuable feedback. I completely agree with your point about the potential confusion caused by the `workdir` parameter.

To avoid confusion with the WORKDIR instruction in Dockerfiles, I suggest renaming the parameter to `prefix-dir` or `target-dir`. This would more accurately reflect its function: specifying the location of files in the target image.

The modified example might look like this:

```
regctl image mod registry.example.org/repo:v1 --create v1-extended \
  --layer-add "dir=path/to/directory,target-dir=/tmp"
```

What do you think of this change? I'll update the documentation accordingly to clearly explain the parameter's purpose and limitations. I'm open to any further suggestions you might have.